src/components/screens/ServiceTiles.js

```diff
@@ -70,15 +85,21 @@ export const ServiceTiles = ({
               <Image source={{ uri: image }} containerStyle={styles.imageContainerStyle} />
             )}
 
-            {!!html && (
+            {!!contentForAbove && (
               <Wrapper>
-                <HtmlView html={html} />
+                <HtmlView html={contentForAbove} />
               </Wrapper>
             )}
 
             <View style={styles.padding}>
-              {!error && <Service data={data} staticJsonName={staticJsonName} />}
+              {(!error || !htmlError) && <Service data={data} staticJsonName={staticJsonName} />}
```

Comment:
how is `htmlError` important here? isn't it just about the first query with `staticJsonName`? so just `error` is still the condition to check?
otherwise i guess this would need a `&&` so in case of no errors at all we want to render, right?

Reply:
you are right, actually we don't need `htmlError` here.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/4a14506f26a9acce9158aeae6f03598b5af9de05